src/aktualizr_info/aktualizr_info_test.cc

```diff
@@ -37,13 +37,13 @@ class AktualizrInfoTest : public ::testing::Test {
 
   virtual void TearDown() {}
 
-  void SpawnProcess(const char* arg = nullptr) {
+  void SpawnProcess(const std::vector<std::string> args = std::vector<std::string>{}) {
     std::future<std::string> output;
     std::future<std::string> err_output;
     boost::asio::io_service io_service;
     int child_process_exit_code = -1;
 
-    if (arg != nullptr) {
+    for (const auto& arg : args) {
```

Comment:
Add the whole `args` `vector` at once, something like

`executable_args.insert(std::end(executable_args), std::begin(args), std::end(args));`

Reply:
Of course! Done.